common/static/js/vendor/jquery.leanModal.js

```diff
@@ -0,0 +1,61 @@
+// leanModal v1.1 by Ray Stone - http://finelysliced.com.au
+// Dual licensed under the MIT and GPL
+
+// Updated to prevent divs with duplicate IDs from being rendered.
+
+(function($) {
+    $.fn.extend({
+        leanModal: function(options) {
+            var defaults = {
+                top: 100,
+                overlay: 0.5,
+                closeButton: null
+            };
+
+            // Only append the overlay element if it isn't already present.
+            if ($("#lean_overlay").length == 0) {
```

Comment:
You're checking if `length` is zero, which assumes the element exists. If it doesn't exist, it'd return `undefined`. So this may cause issues. Did you check to see if this works? It looks like it would always return something other than `0` and therefore would always append a new element.